tox.ini

```diff
@@ -1,7 +1,21 @@
 [tox]
 envlist = py27-django{18,19,110,111}
 
+# This is needed to prevent the lms, cms, and openedx packages from getting
+# installed into site-packages where they can get imported, which is bad
+# because those won't even contain most of the source code.
+skipsdist=True
+
+# The default toxworkdir is in the source tree, but `django-admin
+# compilemessages` cannot handle encountering a .tox directory when it
+# unconditionally traverses the entire directory tree.
+toxworkdir={homedir}/.tox
```

Comment:
This worries me a bit; we could start having environment name collisions if another package starts to do the same thing.  And there is an existing `~/tox` directory, so it'll get tricky to clean out just the files from old edx-platform tox runs without nuking something tox normally puts there.

Why is compilemessages traversing `.tox`?  It looks like it's only supposed to be looking under the entries in the `LOCALE_PATHS` setting.

Reply:
> This worries me a bit; we could start having environment name collisions if another package starts to do the same thing. And there is an existing `~/tox` directory, so it'll get tricky to clean out just the files from old edx-platform tox runs without nuking something tox normally puts there.

Indeed, the filename could be named edxapp-toxenv (a la edxapp-env).  Do we know how `~/tox` gets installed?  Maybe from other people making the same mistake?

> Why is compilemessages traversing `.tox`? It looks like it's only supposed to be looking under the entries in the `LOCALE_PATHS` setting.

There's a new block of code introduced in 1.9 which walks `.` regardless of `LOCALE_PATHS`:

https://github.com/django/django/commit/40f0a84cb151669313faadf857aaddd18d39aaeb#diff-34c8a535d0d99b210470bf9526b48d9bR64